pkg/scheduler/framework/plugins/interpodaffinity/scoring.go

```diff
@@ -207,7 +207,7 @@ func (pl *InterPodAffinity) PreScore(
 func getPreScoreState(cycleState *framework.CycleState) (*preScoreState, error) {
 	c, err := cycleState.Read(preScoreStateKey)
 	if err != nil {
-		return nil, fmt.Errorf("Error reading %q from cycleState: %v", preScoreStateKey, err)
+		return nil, fmt.Errorf("error reading %q from cycleState: %v", preScoreStateKey, err)
```

Comment:
No need for the word `error` in an error. Same for any other occurrence in this PR.

Reply:
You can say "failed to read..." or "cannot read...".

Reply:
`Error` in an error in this PR have been updated with `failed to`, waiting for your review.